relax/data/module.py

```diff
@@ -530,26 +544,36 @@ def load_data(
     
     _validate_dataname(data_name)
 
-    # get data/config urls
+    # get data/config/model urls
     _data_path = DEFAULT_DATA_CONFIGS[data_name]['data']
     _conf_path = DEFAULT_DATA_CONFIGS[data_name]['conf']
+    _model_path = DEFAULT_DATA_CONFIGS[data_name]['model']
     
     data_url = f"https://github.com/BirkhoffG/ReLax/raw/master/{_data_path}"
     conf_url = f"https://github.com/BirkhoffG/ReLax/raw/master/{_conf_path}"
+    model_params_url = f"https://github.com/BirkhoffG/ReLax/raw/master/{_model_path}/params.npy"
+    model_tree_url = f"https://github.com/BirkhoffG/ReLax/raw/master/{_model_path}/tree.pkl"
 
     # create new dir
     data_dir = Path(os.getcwd()) / "cf_data"
     if not data_dir.exists():
         os.makedirs(data_dir)
-    data_path = data_dir / f'{data_name}.csv'
-    conf_path = data_dir / f'{data_name}.json'
+    data_path = data_dir / data_name / 'data.csv'
+    conf_path = data_dir / data_name / 'configs.json'
+    model_path = data_dir / data_name / "model"
+    if not model_path.exists():
+        os.makedirs(model_path)
 
-
-    # download data/configs
+    # download data/configs and trained model
     if not data_path.is_file():
         urlretrieve(data_url, data_path)    
     if not conf_path.is_file():
         urlretrieve(conf_url, conf_path)
+    params_path = os.path.join(model_path, "params.npy")
```

Comment:
Why do you download model params in this function? Why not download model params in `load_pred_model`?

Reply:
Convinient to use DEFAULT_DATA_CONFIGS and call _validate_dataname()

Reply:
I think this logic can potentially lead to bugs. If the user does not load the data through `load_data`, he will encounter an error when calling `load_pred_model` because the model file is not downloaded.

`load_data` should download only the dataset and configs, and `load_pred_model` should download the model and configs (if not already downloaded).